### PR TITLE
Closed internships and deadlines added

### PR DESCRIPTION
**Pull request by anthonydu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ To contribute:
...
 | [Doordash](https://boards.greenhouse.io/doordash/jobs/5309455?gh_jid=5309455)                                                                              | Toronto                                 | Unknown               | Software Engineer, Intern - Toronto (Summer 2024)                                              | Aug 23      |
-| [Vial](https://jobs.lever.co/Vial/574e2485-a3ea-4766-861f-7adba543169a)                                                                                    | Remote                                  | Unknown               | Software Engineer Intern                                                                       | Aug 21      |
+| [Vial](https://jobs.lever.co/Vial/574e2485-a3ea-4766-861f-7adba543169a)                                                                                    | Remote                                  | :lock: Closed :lock:  | Software Engineer Intern                                                                       | Aug 21      |
```
Comment (kanjurer) [inline README.md:48]: Should also remove the link

Comment (anthonydu) [reply README.md]: Fixed! A bit late so might now be a duplication of the newer pull request made by you. But the deadline should still be an addition. Info source was UBC job board.

Comment (jenndryden) [reply README.md]: great catch!

**Comment by jenndryden** [comment]
thanks so much -- this is amazing :D

**Pull request merged**